app/src/main/java/org/piramalswasthya/cho/ui/beneficiary_card/edit/EditBeneficiaryDetailsViewModel.kt

```
@@ -0,0 +1,291 @@
+    init {
+        loadStatusOfWomanMaster()
+    }
+
+    private fun loadStatusOfWomanMaster() {
+        viewModelScope.launch {
+            try {
+                val list = statusOfWomanDao.getAllStatusOfWoman()
+                _statusOfWomanList.value = list
+                // Recompute filtered list in case patient info was already set
+                updateFilteredStatusOfWomanList()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)